server/data/quotes.json

```diff
@@ -755,6 +763,10 @@
     "author": "Turkish Proverb",
     "sentence": "One who speaks only one language is one person, but one who speaks two languages is two people."
   },
+  {
+    "author": "Turkish Proverb",
```

Comment:
This is a [duplicate quote](https://github.com/jayehernandez/letra-extension/blob/08226428a914bb75266e566bad4ad86056bab92e/server/data/quotes.json#L755). Please remove it.

Reply:
Hey @Kvaibhav01 apology from my side. I have done the necessary changes and committed them. Please review. ThankYou.

Reply:
Looks good now. @jayehernandez will take care of this PR now for further actions.